app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if @user.errors.any? %>
+    <div id="error_explanation">
+        <div class="alert alert-danger">
+            <%= t("error.the_form_contains") %>
+            <%= pluralize @user.errors.count, t("error.title") %>.
+        </div>
+        <ul>
+            <% @user.errors.full_messages.each do |msg| %>
+                <li><%= msg %></li>
+            <% end %>
+        </ul>
+    </div>
+<% end %>
```

Comment:
Em setup endline cho vscode nha
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files